src/main/java/edu/ucsb/ucsbcslas/entities/RoomSlot.java

```diff
@@ -0,0 +1,136 @@
+package edu.ucsb.ucsbcslas.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+import java.time.LocalTime;
+import java.time.DayOfWeek;
+
+@Entity
+public class RoomSlot {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotNull
+    @Column(nullable = false)
+    private String location;
+
+    @ManyToOne
+    @JoinColumn(name = "quarter")
+    public ActiveQuarter quarter;
+
+    @Column(nullable = false)
+    private DayOfWeek dayOfWeek;
+
+    @Column(nullable = false)
+    private LocalTime startTime;
```

Comment:
looks like the `OnlineOfficeHours` entity is currently just using a String to represent start time and end time. should that one eventually be updated to also use a LocalTime, and are there any possible drawbacks to using LocalTimes?

Reply:
esp. when we output this data in the CRUD endpoints, will LocalTimes result in a format that's less easily consumable in the frontend than a plain string?

Reply:
I agree it's important that the types for times in all repositories is consistent! Discussed this issue with @btk5h in section and agreed to defer refactoring to a later issue, because it will likely be an involved process.

Reply:
Also reverted the application.properties file now that code coverage is 100% - needed it that way for local testing. Also deferring this to a separate, later issue.